sklearn/neural_network/multilayer_perceptron.py

```diff
@@ -302,9 +301,7 @@ def _init_coef(self, fan_in, fan_out, rng):
             # Use the initialization method recommended by
             # Glorot et al.
```

Comment:
This is it, but it's already in the reference section of the class. I think this is enough.